tkltest/generate/ui/generate_with_labels_analysis.py

```diff
@@ -31,7 +31,9 @@
 
 # names and paths for generated code files
 _POM_FILE = 'pom.xml'
-_CRAWL_PATHS_FILE = 'crawl_paths_addressbook_small.json'
+
+_CRAWL_PATHS_FILE = 'crawl_paths_tmf.json'
```

Comment:
@divtosz why do specific app files/name appear in the code?

Reply:
this is just for testing purposes